pnpm-lock.yaml

```diff
@@ -4,34 +4,32 @@ specifiers:
   '@algolia/client-search': ^4.15.0
   '@changesets/changelog-git': ^0.1.14
   '@changesets/cli': ^2.26.0
-  '@finsweet/docs-theme': ^0.1.2
```

Comment:
I will add this after merging the docs-theme PR https://github.com/finsweet/docs-theme/pull/17